go-controller/pkg/ovn/address_set/address_set.go

```diff
@@ -356,8 +345,6 @@ func (as *ovnAddressSets) GetName() string {
 
 func (as *ovnAddressSets) SetIPs(ips []net.IP) error {
 	var err error
-	as.Lock()
```

Comment:
By removing this, `SetIPs` is now no longer thread-safe. If you think this is OK (which I do), then you should probably document this.

Reply:
@squeed yeah, should be documented. Also agree it should be OK to remove the lock.